### PR TITLE
chore: updated sentry spring boot version

### PR DESCRIPTION
**Pull request by jayanta2018**
TISNEW-5661

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -56,7 +56,8 @@ dependencies {
   testAnnotationProcessor "org.mapstruct:mapstruct-processor:1.3.1.Final"
 
   // Sentry reporting
-  implementation "io.sentry:sentry-spring:1.7.30"
+  implementation "io.sentry:sentry-spring-boot-starter:3.1.1"
+  implementation "io.sentry:sentry-logback:3.1.1"
```
Comment (Judge40) [inline build.gradle:60]: Do you think we should create a property for the version to avoid the potential for mismatch?

Comment (Judge40) [reply build.gradle:60]: See line 43 for camel example.

Comment (jayanta2018) [reply build.gradle:60]: Yeah @Judge40 That's a good idea.
I will do that. Thanks

Comment (jayanta2018) [reply build.gradle]: I have made changes

**Review by Judge40 — APPROVED** [review]

**Pull request merged**